modules/reindex/src/test/java/org/elasticsearch/index/reindex/RoundTripTests.java

```diff
@@ -200,10 +200,10 @@ private void roundTrip(Streamable example, Streamable empty) throws IOException
     }
 
     private Script randomScript() {
-        return new Script(randomSimpleString(random()), // Name
-                randomFrom(ScriptType.values()), // Type
-                random().nextBoolean() ? null : randomSimpleString(random()), // Language
-                emptyMap()); // Params
+        return new Script(randomFrom(ScriptType.values()), // Type
```

Comment:
Instead of comments can you switch these to named variables? That'd be easier to read I think.

I *think* I wrote this code so I'm sorry for asking you to clean if up!